src/components/Purchase/RedMembershipUnavailable.tsx

```diff
@@ -0,0 +1,93 @@
+import { Button, Center, Container, Group, Stack, Text, Title } from '@mantine/core';
+import clsx from 'clsx';
+import { useBuzzCurrencyConfig } from '~/components/Currency/useCurrencyConfig';
+
+import { Meta } from '~/components/Meta/Meta';
+import { NextLink as Link } from '~/components/NextLink/NextLink';
+import { useFeatureFlags } from '~/providers/FeatureFlagsProvider';
+// import classes from '~/pages/pricing/index.module.scss';
+
+interface RedMembershipUnavailableProps {
+  onSelectGreen: () => void;
+  onGoBack: () => void;
+}
+
+export function RedMembershipUnavailable({
+  onSelectGreen,
+  onGoBack,
+}: RedMembershipUnavailableProps) {
+  const features = useFeatureFlags();
+  const { classNames: redClassNames } = useBuzzCurrencyConfig('red');
+  const { classNames: greenClassNames } = useBuzzCurrencyConfig('green');
+
+  return (
+    <>
+      <Meta
+        title="Memberships | Civitai"
+        description="As the leading generative AI community, we're adding new features every week. Help us keep the community thriving by becoming a Supporter and get exclusive perks."
+      />
```

Comment:
Feels like this <Meta> component can be upper on the tree, probably in the page where we call the `RedMembershipUnavailable` component? That way we can also deIndex are add more properties if needed